cdap-standalone/src/main/java/co/cask/cdap/StandaloneMain.java

```diff
@@ -302,6 +317,15 @@ public void startUp() throws Exception {
 
     remoteSystemOperationsService.startAndWait();
 
+    if (previewServer != null) {
+      state = previewServer.startAndWait();
+      if (state != Service.State.RUNNING) {
+        throw new Exception("Failed to start Preview.");
+      }
+      System.out.println("CDAP Preview started successfully.");
```

Comment:
Renamed `getLoggers` to `getTracers` and made similar change in the http handler path as well.